### DISCUSSION TITLE
Fileupload Help

### DISCUSSION THREAD

**Discussion by AxelSalim** [primefaces/community#1442] [PrimeVue Templates] [answered]
Bonjour les amies j'ai un soucis au niveau de fileupload progressbar 
Je voudrais l'utiliser mais j'y arrive pas quelque pourrait-il m'aider ?

**Comment by tugcekucukoglu** [comment]
Can you describe the problem that you've encounter?

**Reply by AxelSalim** [reply]
j'essaye d'utiliser le filupload de primevue mais je n'arrive pas récupérer les fichiers ou ressources que je téléverse 
je sais pas comment le faire

**Comment by tugcekucukoglu** [comment, marked as answer]
You could choose the emit that is most suitable for you. https://primevue.org/fileupload/#api.fileupload.emits

**Reply by AxelSalim** [reply]
En faite je ne comprend pas trop bien son fonctionnement ce serait possible d'avoir d'explication de ta part ?

**Reply by AxelSalim** [reply]
je pourrais avoir un retour de ta part ?
